src/test/java/com/apeshko/springboot/repository/TicketsRepositoryTest.java

```diff
@@ -0,0 +1,40 @@
+package com.apeshko.springboot.repository;
+
+import com.apeshko.springboot.model.Ticket;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.boot.test.autoconfigure.orm.jpa.TestEntityManager;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.test.context.support.AnnotationConfigContextLoader;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@ExtendWith(SpringExtension.class)
+@Tag("integration")
+@DataJpaTest
+class TicketsRepositoryTest {
```

Comment:
that's fine for this task, however I'd say in such simple case you shouldn't test JpaRepository, just because mostly it's not your implementation, so it's not your responsibility to test it

Reply:
@mikalai-amusin agree! Just wanted to play with the in-memory DB setup for testing.